sct.py

```diff
@@ -1117,6 +1119,9 @@ def send_email(test_id=None, test_status=None, start_time=None, started_by=None,
                     "nodes": get_running_instances_for_email_report(test_id, runner_ip)
                 })
         test_results['logs_links'] = list_logs_by_test_id(test_results.get('test_id', test_id))
+        if 'longevity' in job_name:
+            pt_report_urls = search_parallel_timelines_report_urls(test_id=test_results.get('test_id', test_id))
+            test_results['parallel_timelines_report'] = pt_report_urls[0] if pt_report_urls else pt_report_urls
```

Comment:
What false value do you mean? The function `search_parallel_timelines_report_urls` returns list, so I just wanted to check if this list is empty or not. If the list is empty it is ok to pass it as is to `test_results['parallel_timelines_report']` - it will not be used during the jinja template rendering anyway.

Reply:
The following values are considered false in Python:

- `None`.
- `False`.
- Zero of any numeric type. For example, `0` , `0.0` , `0j`.
- Empty sequence. For example, `()` , `[]` , `''` .
- Empty mapping. For example, `{}`
- objects of Classes which has `__bool__()` or `__len()__` method which returns `0` or `False`.

In your case it's create a mess: type of `test_results['parallel_timelines_report']` is `str | list` effectively and it's a list in the case of empty `pt_report_urls` only. It's better to make it `str | None` by explicitly set it to `None` in case of empty `pt_report_urls`. Does it make sense?

Reply:
Yes, I got it. Made in `None`.